### DISCUSSION TITLE
PatchTool 6.3 1/12 漢化器更新

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#277] [Announcements]
※並沒有更新漢化文本和修bug，沒有需求者可以跳過這個release。

漢化器新增選擇是否替換字體和文本的選項，預設皆為開啟。

若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。

關於高畫質字體包可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

![image](https://user-images.githubusercontent.com/34565875/211917814-25da2744-fd31-4fd6-ad4e-eb683feab43f.png)

檔案：

- FFXIVChnTextPatch-20230112.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：高畫質字體包，請取代原本的resource/font資料夾。
- rawexd-zhcn-20230112.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.0/backup-20230111.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.0.0'>PatchTool 6.3 1/12 漢化器更新</a>.</em>

**Comment by hyhv80435** [comment]
您好 之前沒漢化過 第一次使用這個漢化包 步驟都沒有錯但是都沒有漢化成功 我是國際服日服的玩家 不知道哪裡出問題 
下載 FFXIVChnTextPatch-20230112.rar
打開 FFXIVChnTextPatchGP
遊戲路徑選FINAL FANTASY xiv - A Realm Reborn
檔案語言選 CSV 原始語言日文 確認 
還是失敗 請問我操作上哪邊有失誤嗎? 謝謝

**Reply by GpointChen** [reply]
失敗是指沒辦法跑漢化還是跑完遊戲沒有變成中文？

**Comment by weicheun** [comment]
您好 之前沒漢化過 第一次使用這個漢化包 我的CSV開不起來都會出現( 未知格式或錯誤 )請問要怎麼處理?

**Reply by GpointChen** [reply]
這可能跟你的作業系統或環境有關，原因很多我無法斷定；不過正常情況下，你不需要額外開啟CSV檔即可進行漢化，參考readme.md操作即可。

如果你想要自行編輯CSV檔，請注意不要使用excel進行編輯，因為excel儲存CSV時並不是儲存成純文本的格式。

**Reply by weicheun** [reply]
你好 請問大大可以用遠端幫我看一下狀況嗎?

**Reply by GpointChen** [reply]
你現在是打算做什麼，卡在哪一步？我覺得你可以截個圖方便我了解情況。

**Comment by xxcornss** [comment]
8樓後半boss對話會是空白
![image](https://user-images.githubusercontent.com/122058757/212563239-ebd82b40-3a7c-45a8-a386-72fcbeb4dd0e.png)

**Comment by Chinmokukun** [comment]
用高画质字体包会出这个报错
![213009490-836c91ff-94a0-4544-b4ad-77bd3a6c9361](https://user-images.githubusercontent.com/123024134/213292909-fe5b83d9-39d4-44ba-8421-33f65d3ed857.png)

**Reply by GpointChen** [reply]
目前看來有一部分人使用高畫質包會報錯，待後續確認原因。

**Reply by Chinmokukun** [reply]
> 目前看來有一部分人使用高畫質包會報錯，待後續確認原因。

了解

**Comment by Bos-hunting** [comment]
希望有6.31的兼容版，進入新絕的時候偶爾會有閃退或者卡頓的錯誤。十分感謝作者的努力，感恩。

**Reply by GpointChen** [reply]
已經更新至6.31，不過我體感上沒有發現有哪裡可能導致絕本閃退；如果仍有問題歡迎再回報。

**Comment by ksledu** [comment]
> 關於高畫質字體包可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

請問[font-custom-20230111.rar 這個字體包是4K 分辨率的字體嗎？因為我自己有從別處拿到的4K字體包，我對比了一下與你高質量字體包的區別，除了您大廳字體的體積比我的容量大之外，其他的字體的體積都比我的4K字體容量要小。 
PS：我自己的4K字體包在遊戲里顯示沒有問題。 謝謝大大的貢獻。

**Reply by GpointChen** [reply]
可能是我有重複使用部分同大小字型吧，所以沒有占用那麼多空間？我也不確定。

**Reply by ksledu** [reply]
謝謝作者回復

**Comment by okami2223** [comment]
不好意思，請教下作者~
不知道是不是我電腦有遺失字體還是怎麼樣，總覺得用了4K字體跟沒4K字體都是一樣模糊
![image](https://user-images.githubusercontent.com/44223291/215521595-bdfc5fd5-ddbf-4740-a533-c6b0225f134d.png)

**Reply by GpointChen** [reply]
小字體的差異可能不大，通常是比較大的字才能看出差異。

預設字體包：

![image](https://user-images.githubusercontent.com/34565875/215623607-964a2858-a0ca-4d17-80dc-2a29423a9a0b.png)
![image](https://user-images.githubusercontent.com/34565875/215625318-4a14f133-71e8-4a2a-afff-79ba371cbd10.png)


高畫質字體包1：

![image](https://user-images.githubusercontent.com/34565875/215624575-35f75ab4-736f-450c-bb5d-0fc74f588018.png)
![image](https://user-images.githubusercontent.com/34565875/215624773-2b616c4c-78e9-4fd4-9c12-151950873057.png)

**Reply by okami2223** [reply]
回報一下，今日驅動換了最新的，字體恢復了正常高清了
更新前使用1/31高清字體仍模糊，更換驅動後即正常顯示
![image](https://user-images.githubusercontent.com/44223291/215940059-aa05f73f-730d-458f-ab6c-4748835ec973.png)
